### PR TITLE
CSV bulk upload also receive the location of the sounds

### PR DESCRIPTION
**Pull request by andrebola**
As described in issue #829 now the command csv_bulk_upload receives the
location of the sounds

When I was testing the changes I couldn't make it work because of the line  https://github.com/MTG/freesound/blob/master/sounds/management/commands/csv_bulk_upload.py#L119

which (I think) is incorrect, I also changed that line removing `self.`

### CODE REVIEW

File: sounds/management/commands/csv_bulk_upload.py [hunk 1]
```diff
@@ -116,14 +122,15 @@ def handle(self, *args, **options):
             user = User.objects.get(username=usernamef)
 
             # 1 create dir and move sound to dir
-            directory = self.user.profile.locations()['uploads_dir']
```
Comment (ffont) [inline sounds/management/commands/csv_bulk_upload.py]: wow this was a bug right? I mean the command does not really work (before applying this PR) right?

Comment (andrebola) [reply sounds/management/commands/csv_bulk_upload.py]: I think so, it didn't work in my local.

Comment (ffont) [reply sounds/management/commands/csv_bulk_upload.py]: Oh I saw the comment in description now, nevermind.

**Comment by ffont** [comment]
Great, I'll merge master into it to pass the tests and merge the PR.

**Pull request merged**